airflow/utils/log/file_task_handler.py

```diff
@@ -187,10 +187,14 @@ def _read(self, ti, try_number, metadata=None):
                 response.encoding = "utf-8"
 
                 if response.status_code == 403:
-                    log += "*** !!!! Please make sure that all your webservers and workers have" \
-                           " the same 'secret_key' configured in 'webserver' section !!!!!\n***"
-                    log += "*** See more at https://airflow.apache.org/docs/apache-airflow/" \
-                           "stable/configurations-ref.html#secret-key\n***"
+                    log += (
+                        "*** !!!! Please make sure that all your webservers and workers have"
+                        " the same 'secret_key' configured in 'webserver' section !!!!!\n***"
+                    )
+                    log += (
+                        "*** See more at https://airflow.apache.org/docs/apache-airflow/"
+                        "stable/configurations-ref.html#secret-key\n***"
+                    )
```

Comment:
This isn't related to this PR

Reply:
I broke it and just fixed it in main (bad Jarek!)

Reply:
Needs rebase